PowervRA/Functions/Private/xRequiresVAMI.ps1

```diff
@@ -0,0 +1,61 @@
+﻿function Global:xRequiresVAMI {
```

Comment:
You should be able to drop the Global: part of this now, since we have restructured the module to make Private functions actually private.